File: euler/p1/p1.go
package main

import "fmt"

func isP1(x int) bool {
	return x % 3 == 0 || x % 5 == 0;
}

func main() {
	var sum = 0;
	for i := 0; i < 1000; i++ {
		if isP1(i) {
			sum += i;
		}
	}
	fmt.Printf("%d\n", sum);
}

File: euler/p1/p1.c
#include <stdio.h>

int is_p1(int n) {
	return n % 3 == 0 || n % 5 == 0;
}

int main() {
	int sum = 0;
	for (int i = 0; i < 1000; i++) {
		if (is_p1(i)) {
			sum += i;
		}
	}
	printf("%d\n", sum);
	return 0;
}
